data/json/items/fake.json

```diff
@@ -179,6 +179,14 @@
     "name": { "str": "jointer" },
     "copy-from": "fake_appliance_tool"
   },
+  {
+    "id": "catalytic_cracking_reactor_tool",
+    "type": "TOOL",
+    "copy-from": "fake_appliance_tool",
+    "name": { "str": "catalytic cracking reactor", "str_pl": "catalytic cracking reactors" },
+    "charges_per_use": 50,
```

Comment:
Is this an ok amount?

Reply:
I am not sure why it would use any? It does not use any (directly) in the existing form, why should the appliance form?

Reply:
A real one I believe ought to use some for circulation of catalyst in the cases where the chemset creates the heat, otherwise heating too.
I realise now that correcting that might be out of scope for making it an appliance?

Reply:
Actually testing shows that this doesn't draw any power when used, only makes it require that many available to be able to be used.